bin/setup

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require "pathname"
+require "fileutils"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"]
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
```

Comment:
This sets ENV["RAILS_ENV"] to ENV["PASSENGER_APP_ENV"] no matter what. So if PASSENGER_APP_ENV is not set, `RAILS_ENV` will exist and be set to `""`. Seems more appropriate that if RAILS_ENV or PASSENGER_APP_ENV are not set, we don't create the environment at all.

```ruby
ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"] if ENV["PASSENGER_APP_ENV"]
ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"] if ENV["PASSENGER_BASE_URI"]
```